### PR TITLE
New cards for 'Heal Combo' deck

### PR DESCRIPTION
**Pull request by AlexNisnevich**
#497.

### CODE REVIEW

File: src/common/constants.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const LOG_SOCKET_IO = false;
 export const KEEP_DECKS_UNSHUFFLED = false;
 export const DISABLE_TURN_TIMER = false;
 export const DISABLE_AI = false;
-export const DISPLAY_HEX_IDS = false;
+export const DISPLAY_HEX_IDS = true;
```
Comment (jacobnisnevich) [inline src/common/constants.js]: Is this the constant to display the hex IDs on the board? If so why is it true now?

File: test/store/cards.spec.js [hunk 2]
```diff
@@ -14,10 +14,15 @@ describe('Built-in cards', () => {
...
-      abilities.forEach(ability => {
-        executeCmd(state, ability, dummyCurrentObj);
-      });
+      try {
+        const abilities = (card.abilities || []).concat(card.command || []);
+        abilities.forEach(ability => {
+          executeCmd(state, ability, dummyCurrentObj);
+        });
+      } catch (err) {
+        console.error(`Error executing ability for ${card.name} card!`);  // eslint-disable-line no-console
```
Comment (jacobnisnevich) [inline test/store/cards.spec.js]: Do we have this comment everywhere we have a `console.error`? Maybe we can just enable `console.error` and not `console.log` if that's possible?

File: src/common/constants.js (hunk 1 above)
Comment (AlexNisnevich) [reply src/common/constants.js]: oops

File: test/store/cards.spec.js (hunk 2 above)
Comment (AlexNisnevich) [reply test/store/cards.spec.js]: Yeah, let me try that

**Pull request merged**